forum/backends/mongodb/api.py

```diff
@@ -1581,19 +1581,25 @@ def get_course_id_by_thread_id(thread_id: str) -> str | None:
         """
         Return course_id for the matching thread.
         """
-        thread = CommentThread().get(thread_id)
-        if thread:
-            return thread.get("course_id")
+        try:
+            thread = CommentThread().get(thread_id)
+            if thread:
+                return thread.get("course_id")
```

Comment:
Shouldn't this be: `return thread["course_id"]`? (likewise below)

Reply:
In case of standalone threads, thread don't have course_id.
Just to avoid key error and returning None. Although we don't have standalone thread now as far as I know.

Reply:
Please add a comment to explain that. (and similar below)

Reply:
Done